.gitignore

```diff
@@ -1,4 +1,30 @@
-.idea/
+*.class
+*.idea
+.settings
+.project
+*.sass-cache
+target
+.java-version
+*.releaseBackup
+release.properties
+README.html
+
+# Package Files #
+*.jar
+*.war
+*.ear
 *.iml
-target/
-**\.DS_Store
+
+.settings/*
+api/.classpath
+api/.project
+api/.settings/*
+api/target/*
```

Comment:
**target** [line 6](https://github.com/Bahmni/bahmni-offline-sync/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R6) takes care of this one